.helix/languages.toml

```diff
@@ -0,0 +1,19 @@
+[language-server.mojo-lsp]
```

Comment:
**Suggestion** Did you mean to check this in?  If so, can you split this out into a separate PR?  This file won't be managed by copybara, so when I import this PR internally, it would never make its way back out.  The fileset managed by copybara is explicit and not all of the files in the mojo repo right now.

Reply:
I imported the PR right now, but it won't include this file when I land it FYI.